squad_client/commands/submit.py

```diff
@@ -0,0 +1,165 @@
+import json
+import logging
+import os
+import yaml
+from squad_client.shortcuts import submit_results
+from squad_client.core.command import SquadClientCommand
+
+
+logger = logging.getLogger()
+
+
+class SubmitCommand(SquadClientCommand):
+    command = "submit"
+    help_text = "submit results to SQUAD"
+
+    def register(self, subparser):
+        parser = super(SubmitCommand, self).register(subparser)
+        result_group = parser.add_mutually_exclusive_group()
+        result_group.add_argument(
+            "--results",
+            help="File with test results to submit. Max 5MB. JSON and YAML formats are supported",
+        )
+        result_group.add_argument("--result-name", help="Single result name")
+        parser.add_argument(
+            "--result-value",
+            help="Single result output (pass/fail)",
+            choices=["pass", "fail"],
+        )
+        parser.add_argument(
+            "--metrics",
+            help="File with metrics(benchmarsk) to submit. Max 5MB. JSON and YAML formats are supported",
+        )
+        parser.add_argument(
+            "--metadata",
+            help="File with metadata to submit. Max 5MB. JSON and YAML formats are supported",
+        )
+        parser.add_argument(
+            "--attachments",
```

Comment:
I get `TypeError: 'NoneType' object is not iterable` when not passing attachments, perhaps `default=[]` would fix it? I guess default is `None`

Reply:
ack, done.